homeassistant/components/kodi/media_player.py

```diff
@@ -281,11 +281,26 @@ async def wrapper(obj, *args, **kwargs):
         except jsonrpc_base.jsonrpc.TransportError as exc:
             # If Kodi is off, we expect calls to fail.
             if obj.state == STATE_OFF:
-                log_function = _LOGGER.info
+                log_function = _LOGGER.debug
             else:
                 log_function = _LOGGER.error
             log_function(
-                "Error calling %s on entity %s: %r", func.__name__, obj.entity_id, exc
+                "Error calling %s on entity %s: %r",
+                func.__name__,
+                obj.entity_id,
+                exc
+            )
+        except jsonrpc_base.jsonrpc.ProtocolError as exc:
```

Comment:
Maybe we should just catch both types of errors in one clause? The two clauses are practically identical

Reply:
when fixing code from other people, I try to change as little as possible

Reply:
Makes sense, but wouldn't adding an error type to `except` be the minimal change? :)

Reply:
You can still differentiate even with one clause - the log will have the type of error in it

Reply:
you are right

Reply:
you are right